src/net_processing.cpp

```diff
@@ -1992,77 +2003,80 @@ static void ProcessHeadersMessage(CNode& pfrom, CConnman& connman, ChainstateMan
     return;
 }
 
-void static ProcessOrphanTx(CConnman& connman, CTxMemPool& mempool, std::set<uint256>& orphan_work_set, std::list<CTransactionRef>& removed_txn) EXCLUSIVE_LOCKS_REQUIRED(cs_main, g_cs_orphans)
+/**
+ * Reconsider orphan transactions after a parent has been accepted to the mempool.
+ *
+ * @param[in]      connman          Connection manager
+ * @param[in]      mempool          Mempool
+ * @param[in/out]  orphan_work_set  The set of orphan transactions to reconsider. Generally only one
```

Comment:
after this PR, isn't it a max of one orphan that will be reconsidered on each run of the function?

Reply:
Yes. Fixed!